specifyweb/frontend/js_src/lib/queryresultstable.js

```diff
@@ -10,11 +10,29 @@ var ScrollResults = require('./scrollresults.js');
 var QueryResults  = require('./queryresults.js');
 const queryText = require('./localization/query').default;
 const commonText = require('./localization/common').default;
+const dataModelStorage = require('./wbplanviewmodel').default;
```

Comment:
Also here. Use `domian.getTreeDef` instead. That way when that gets refactored, this will get fixed.